tests/MongoDB.Driver.Tests/MongoCredentialTests.cs

```diff
@@ -91,10 +95,10 @@ public void TestEquals()
         [Fact]
         public void TestPassword()
         {
-            var credentials = MongoCredential.CreateMongoCRCredential("database", "username", "password");
 #pragma warning disable 618
+            var credentials = MongoCredential.CreateMongoCRCredential("database", "username", "password");
             Assert.Equal("password", credentials.Password);
-#pragma warning restore
```

Comment:
618 was missing so I brought it back.

Reply:
In the past we've tended to restore all warnings, so it was missing 618 on purpose. So you'll find tons of lines like this one in the code base.

But it's not a bad idea to restore JUST the warning you disabled.

I guess the minor risk we were protecting against was an error like this one:

    #pragma warning disable 618
    // ...
    #pragma warning restore 619